Library/SharedFunctions.swift

```diff
@@ -661,7 +661,11 @@ public func isRewardDigital(_ reward: Reward?) -> Bool {
     .isAny(of: Reward.Shipping.Preference.none)
 }
 
-public func estimatedShippingText(for reward: Reward, selectedShippingRule: ShippingRule) -> String {
+public func estimatedShippingText(
+  for reward: Reward,
+  project: Project,
+  selectedShippingRule: ShippingRule
+) -> String {
```

Comment:
I needed to update this method to accept the current Project. This allowed me to use the `Format.currency()` helper which gives us the correct currency symbol and formatting that we need.